hugo/layouts/langium/list.html

```diff
@@ -4,9 +4,19 @@
 <div class="flex flex-wrap gap-10 justify-center mt-5 mx-5 "> 
 {{range where .Pages.ByWeight ".Params.draft" false}}
     <div class="rounded-lg border shadow-md bg-gray-800 border-gray-600 w-[300px] overflow-hidden self-stretch text-white">   
+        {{ if .Params.url }}
         <a href="{{.Params.url}}" class="self-stretch">
+        {{ else if .Params.externalUrl }}
+        <a href="{{.Params.externalUrl}}" onclick="return confirm('You will be redirected to a external website: \n{{.Params.externalUrl}}\n\nAre you sure you want to continue?')" target="_blank" class="self-stretch">
```

Comment:
I think this confirmation is not necessary. The icon and `target="_blank"` already emphasizes that we're going somewhere else.